client/src/search.tsx

```diff
@@ -236,10 +244,7 @@ function InnerSearchNavigateWidget(props: InnerSearchNavigateWidgetProps) {
     // overlaying search results don't trigger a scroll.
     const limit = window.innerHeight < 850 ? 5 : 10;
 
-    const q: string[] = inputValue
-      .toLowerCase()
-      .split(" ")
-      .map((s) => s.trim());
+    const q: string[] = splitQuery(inputValue);
```

Comment:
I would skip `: string[]`, since it can be inferred.

Reply:
```suggestion
    const q = splitQuery(inputValue);
```